index.js

```diff
@@ -1,16 +1,15 @@
-const assert = require("assert");
-const removeSlash = require("remove-trailing-slash");
-const looselyValidate = require("@segment/loosely-validate-event");
-const serialize = require("serialize-javascript");
-const axios = require("axios");
-const axiosRetry = require("axios-retry");
-const ms = require("ms");
-const { v4: uuid } = require("uuid");
-const md5 = require("md5");
-const isString = require("lodash.isstring");
-const bunyan = require("@expo/bunyan");
-
-const version = require("./package.json").version;
+const assert = require('assert');
+const removeSlash = require('remove-trailing-slash');
+const looselyValidate = require('@segment/loosely-validate-event');
+const originalFetch = require('node-fetch');
```

Comment:
wrap node fetch with retry logic

Reply:
Name this just fetch
```suggestion
const fetch = require('node-fetch');
```